ImperatorToCK3/Data_Files/blankMod/output/localization/english/CONVERTER_dejure_titles_static_l_english.yml

```
@@ -0,0 +1,28 @@
+﻿l_english:
+ e_gaul: "Gaul"
+ k_armorique: "Armoricia"
+ k_aquitane_gaul: "Aquitane"
+ k_belgia: "Belgia"
+ k_celtia_gaul: "Celtia Gual"
```

```suggestion
 k_celtia_gaul: "Celtia Gaul"
```
@IohannesIohannium does this name make sense?